src/components/Intro.vue

```diff
@@ -144,11 +144,41 @@ export default {
   },
   computed: {
     ...mapGetters(['isQuizLoaded']),
-    incrementPageDisabled() {
-      if (this.page === 4) {
-        return !this.isQuizLoaded;
+    pageTransitions() {
+      return {
+        1: {
+          canGoNext() {
+            return this.userInfo.name !== '';
+          },
+        },
```

Comment:
W sumie w 2 można sprawdzać czy to Int, bo można tam dać np spację

Reply:
@fraunos dodane